website/src/css/custom.css

```diff
@@ -1,6 +1,35 @@
 /* Stick any global custom CSS in this file */
 
 /* You can override the default Infima variables here. */
+@layer theme, base, components, utilities;
+@import "tailwindcss/theme.css" layer(theme);
+@import "tailwindcss/utilities.css" layer(utilities);
+
+@config "../../tailwind.config.js";
```

Comment:
The `corePlugins` Javascript option is not supported anymore, so I don't import `base` from Tailwind here to disable preflight.

https://tailwindcss.com/docs/upgrade-guide#using-a-javascript-config-file